client/app/components/Input/__tests__/InputCheckboxGroup.spec.jsx

```diff
@@ -35,44 +36,42 @@ describe('InputCheckboxGroup', () => {
     jest.spyOn(window, 'alert');
   });
 
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
   describe('has no required prop', () => {
     it('does not call hasError prop when all checkboxes are unchecked', () => {
-      const wrapper = mount(
+      render(
         <InputCheckboxGroup checkboxes={checkboxes} hasError={someEvent} />,
       );
-      act(() => {
-        wrapper
-          .find(`input[name="${name}"][type="checkbox"]`)
-          .prop('onChange')({ currentTarget: { checked: false } });
-      });
-      act(() => {
-        wrapper.find(`input[name="${nameTwo}"]`).prop('onChange')({
-          currentTarget: { checked: false },
-        });
-      });
+      const checkbox = screen.getByRole('checkbox', { name: label });
+      const otherCheckbox = screen.getByRole('checkbox', { name: labelTwo });
+
+      // toggle both checkboxes false
+      userEvent.click(checkbox);
+      userEvent.dblClick(otherCheckbox);
+
       expect(window.alert).not.toHaveBeenCalled();
     });
   });
 
   describe('has required prop', () => {
     it('does calls hasError prop when all checkboxes are unchecked', () => {
-      const wrapper = mount(
+      render(
         <InputCheckboxGroup
           checkboxes={checkboxes}
           hasError={someEvent}
           required
         />,
       );
-      act(() => {
-        wrapper
-          .find(`input[name="${name}"][type="checkbox"]`)
-          .prop('onChange')({ currentTarget: { checked: false } });
-      });
-      act(() => {
-        wrapper.find(`input[name="${nameTwo}"]`).prop('onChange')({
-          currentTarget: { checked: false },
-        });
-      });
+      const checkbox = screen.getByRole('checkbox', { name: label });
+      const otherCheckbox = screen.getByRole('checkbox', { name: labelTwo });
+
+      // toggle both checkboxes false
+      userEvent.click(checkbox);
+      userEvent.dblClick(otherCheckbox);
```

Comment:
Can this just be a regular click?

Reply:
Oh right I think I should maybe just leave `otherCheckbox` alone, since it has an initial value of `false` anyways. This double click essentially just toggles it on and off again 😅 I think I transcribed the former test too literally, where it simulated the `onChange` prop firing for a `false` value. Let me change it so that the test just assumes `otherCheckbox` is false to begin with.